### ISSUE TITLE
How to export a wordcloud2 output as an high-resolution image?

### ISSUE THREAD

**Issue by FV83** [Lchiffon/wordcloud2#49]
I have created a nice word cloud with wordcloud2 but the exported png file has a bad resolution when I print it (on a big A3 sheet) - same when I zoom in a bit too much on the screen.
To save the output as an image, I used the "Export/save as image" in the Rstudio viewer.
I also tried via saveWidget (save as html) and webshot (convert html to png) but the resolution is also low.
I have tried with grDevices::png or grDevices::tiff, where you can control for the resolution, but I get a blank image (works with wordcloud but not wordcloud2).
Is there another possibility I should try?

**Comment by courageup** [comment]
I had the same issue. Hope it could be fixed/enhanced soon.

**Comment by ldlpdx** [comment]
Having similar issues. Using a custom mask for mine, but I can get it to work within webshot only if I don't use the custom mask. With the custom mask, it's completely blank.

**Comment by patkelly3189** [comment]
Having the same issue. Anyone come up with a solution? I'm trying to use this wordcloud as a figure in a publication, so unless I can get appropriate resolution it's a no-go.

**Comment by xiaofan1991** [comment]
Having the same issue. First, I can't export vectorgram file such as eps, svg and emf. Second, words failed to fill in the custom mask.

**Comment by imanojkumar** [comment]
i think none is working to resolve this issue... 
Exporting WordCloud2 plot into high resolution png/jpg file (such as 1280x800 size)

**Comment by krlmlr** [comment]
Solved by #8?

**Comment by hyonggu-hwang** [comment]
Going through the same problem in 2020. Could anyone please share if this issue has been resolved?

**Comment by reblake** [comment]
No, this issue hasn't been resolved.  Even using the twitter mask and the demo data from the package, I can't export the image created in the RStudio plotting window.  It is really frustrating.  The code I tested with is here: 
```
figPath2 <- system.file("examples/t.png", package = "wordcloud2")
tw <- wordcloud2(data = demoFreq, figPath = figPath2, color = "random-light", shuffle = FALSE)
tw
saveWidget(tw,"tmp.html", selfcontained = F)
webshot("tmp.html","test-cloud.pdf", delay = 5, vwidth = 480, vheight = 480)
```

**Comment by Erdbeergeist** [comment]
Same problem, using webshot produces a blank image ...

**Comment by annahuynhly** [comment]
I have a makeshift solution (surprised no one mentioned it here... I experimented myself.)

Essentially, increase the `size` argument in the `wordcloud2` and then save the image with increased width and height.

Here's an example of the code, assuming you download a `.png`. You will need to replace the `data` argument with whatever you are using (not giving a reproducible example since the data I'm using is supposed to be private):

```
library(wordcloud2)
library(webshot)
library(htmlwidgets)

webshot::install_phantomjs()
mygraph <- wordcloud2(data = word_freq, size = 3) # change size here

saveWidget(mygraph,"tmp.html",selfcontained = F)
webshot("tmp.html","fig_1.png", delay =5, vwidth = 1280, vheight=800) #ensuring high res
```
